app/controllers/users_controller.rb

```diff
@@ -1,17 +1,24 @@
 class UsersController < ApplicationController
   def show
-      @user = User.find(params[:userID])
+      @user = User.find(params[:id])
+      @faveitems = Array.new
```

Comment:
Will update once Noah's PR is merged

Reply:
Sounds good, once mine is merged you should just have to use User.favorites